content/applications/websites/livechat/overview/get_started.rst

```diff
@@ -2,82 +2,200 @@
-interacting with you. Remember: your customers want to talk to you, so let's make it easy.
+Odoo *Live Chat* allows users to communicate with website visitors in real time. Potential leads
+can be qualified for their sales potential, support questions can be answered quickly, and issues
+can be directed to the appropriate team for further investigation or follow up. *Live Chat* also
+provides the opportunity for instant feedback from customers.
 
-Set up
-======
+Enabling live chat
+==================
 
-Once *Live Chat* is installed on your database, if your website was created with Odoo, the
-application is automatically added to it. All that is left to do is to go to
-:menuselection:`Website --> Configuration --> Settings --> Live Chat`.
+In order to enable :guilabel:`Live Chat`, the application needs to be installed. This can be done
+in one of two ways.
 
-.. image:: get_started/livechat_settings.png
+- Go to :menuselection:`Dashboard --> Apps --> Live Chat` and click :guilabel:`Install`.
+
+- In the :guilabel:`Website` application, go to :menuselection:`Configuration --> Settings`, scroll
+  to the :guilabel:`Email & Marketing` section, and check the box next to :guilabel:`Live Chat`.
+  :guilabel:`Save` any changes.
+
+.. image:: get_started/live-chat-enable-setting.png
    :align: center
-   :height: 300
    :alt: View of the settings page and the live chat feature for Odoo Live Chat
 
-Select the channel to be linked to your website or create one on the fly.
+A :guilabel:`Live Chat Channel` will be created by default and automatically selected in the
+drop-down.
+
+Create a new live chat channel
+==============================
 
-.. image:: get_started/channels.png
+To create a new :guilabel:`Live Chat Channel`, go to :menuselection:`Dashboard --> Live Chat -->
+New`. Enter the name of the new channel in the :guilabel:`Channel Name` field.
+
+.. image:: get_started/live-chat-open-channel.png
    :align: center
-   :alt: View of a live chat channel form for Odoo Live Chat
-
-| For both scenarios, under:
-| - **Operators**: add agents to respond to the chat requests. Add as many as you like, and keep in
-  mind that operators that do not show any activity in Odoo for more than 30min are considered
-  disconnected.
-| - **Options**: set the default text to be shown on the live chat button; an automated welcome
-  message to be seen by visitors when a conversation is initiated, and the text that prompts the
-  user to initiate a chat.
-| - **Channel Rules**: choose an action for a given URL, and/or per country. In the example below,
-  the chat window automatically pops-up 3 seconds after users (from any country) land on the
-  contact us page.
-
-.. image:: get_started/rules.png
+   :alt: View of a live chat channel form for Odoo Live Chat.
+
+To configure the remaining tabs, follow the steps below.
+
+Operators
+---------
+
+*Operators* are the users who will respond to live chat requests from customers. The user who
```

Comment:
Because in this case I am talking about the role "operators" in the abstract, and not something specific to the UI, I am not going to add the guilabel.